lib/components/viewers/stop-viewer.js

```diff
@@ -285,6 +285,15 @@ class StopViewer extends Component {
           <FormattedMessage id="components.StopViewer.noStopsFound" />
         </div>
       )
+      /* If geometries are available (and are not a point) and stop times 
+         are not, it is a strong indication that the stop is a flex stop.
+
+         The extra checks stopData are needed to ensure that the message is 
+         not shown while stopData is loading
+         */
+      if (stopData && stopData.geometries && stopData.geometries.geoJson?.type !== 'Point') {
+        contents = <div><FormattedMessage id="components.StopViewer.flexStop" /></div>
```

Comment:
For some reason this is showing an unnecessary scrollbar (on both Chrome & Firefox). Maybe because of a site-wide CSS `line-height` defined high up... Try setting `line-height` to `normal` on this `<div>` to hide the scrollbar. (An inline style with comment should do.)

Reply:
Nice catch and thank you for providing the solution! Worked great. Implemented in 6df872e51caa9d562533278d8da5122236219c53